examples/suit_update/Makefile

```diff
@@ -61,12 +61,14 @@ QUIET ?= 1
 
 ifeq (,$(filter native,$(BOARD)))
   GNRC_NETIF_NUMOF := 2
-  USEMODULE += ethos
-  USEMODULE += gnrc_uhcpc
+  ifeq (local, $(SUIT_MODE))
```

Comment:
I don't feel this a correct representation of the choice base. I can perfectly use the http server and ethos.